react/AsidedLayout/AsidedLayout.less

```diff
@@ -0,0 +1,21 @@
+@import (reference) "~seek-style-guide/theme";
+
+@media only screen and (min-width:740px) {
```

Comment:
Yeah, unpacking my comment would be "probably meaningful number that *looks* like magic and should use an appropriately named variable"